### DISCUSSION TITLE
Error in languages.toml

### DISCUSSION THREAD

**Discussion by Octagon3** [helix-editor/helix#11070] [Troubleshooting]
### Summary

The command for C# language server is omnisharp, not OmniSharp. 

### Reproduction Steps

 

I tried this:

1. `hx --health c-sharp`

I expected this to happen:

Proper configuration reported

Instead, this happened:

OmniSharp is not on PATH

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Linux

### Terminal Emulator

Console

### Installation Method

Installed

### Helix Version

24.3

**Comment by the-mikedavis** [comment]
It depends on how you install it. Nix for example installs [an `OmniSharp` binary](https://github.com/NixOS/nixpkgs/blob/7dca15289a1c2990efbe4680f0923ce14139b042/pkgs/development/tools/omnisharp-roslyn/default.nix#L53). The upstream seems to default to creating the binary as `OmniSharp` as well https://github.com/OmniSharp/omnisharp-roslyn
